apiserver/application/application.go

```diff
@@ -339,6 +360,100 @@ func (api *API) Update(args params.ApplicationUpdate) error {
 	return nil
 }
 
+// UpdateApplicationSeries updates the application series.  Series for
+// subordinates updated too.
+func (api *APIv5) UpdateApplicationSeries(args params.ApplicationUpdateSeries) error {
+	if err := api.checkCanWrite(); err != nil {
+		return err
+	}
+	if !args.Force {
+		if err := api.check.ChangeAllowed(); err != nil {
+			return errors.Trace(err)
+		}
+	}
+
+	app, err := api.backend.Application(args.ApplicationName)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if args.Series == "" {
+		return params.Error{
+			Message: "series missing from args",
+			Code:    params.CodeBadRequest,
+		}
+	}
+	if err = verifySupportedSeries(app, args.Series, args.Force); err != nil {
+		if params.IsCodeIncompatibleSeries(err) {
+			err = errors.Annotatef(err, "series %s is not supported by the application in use. Use --force to override.", args.Series)
```

Comment:
wrapped appropriate errors in common.ServerError.  Removed checks highlighted in comment.